README.md

```diff
@@ -0,0 +1,23 @@
+# AEM PWA Studio Extensions
+
+This projects is a collection of AEM extensions for [PWA Studio](https://pwastudio.io):
+
+1. [AEM Blog PWA Extension](./aem-cfm-blog-extension) is an extension that allows you to hook a blog to your PWA Studio app.
+2. [AEM CIF Product Page Extension](./aem-cif-product-page-extension) is an extension that allows you decorate the product and category pages with content from an AEM content fragment
+
+For details about using the extension please refer to the extension's README file. For general instructions on how to run them locally, read below:
+
+## Running the extensions locally
+
+Currently, there are no released versions of these extensions as they are merely for demo. If you want to run them locally, you need to do the following:
+
+-   go to the source folder of the extension and run `yarn link`.
```

Comment:
`yarn link` behaves differently than `yarn add --dev file:../extension` to a point where I would actually not recommend using it other than development.

When using `yarn link`, the extensions need to provide some dependencies as dev dependencies (apollo, react, babel, magento build tools, etc.) otherwise the build fails. If you decide to keep using `yarn link`, these dependencies need to be added to the blog post extension.

Reply:
The strange this is, for the PDP extension I only had to list those as `peerDependencies` - the `devDependencies` are there because of the unit tests. 
Of course, this works in conjunction with [this piece of code](https://github.com/dplaton/aem-pdp-decorator/blob/main/webpack.config.js#L96) which ensures that the peer dependencies are treated like they should.

Reply:
I'm fine with both actually :) Just had issues when installing both extensions at the same time. Could you please double check that his still works?